modules/installation-configuration-parameters.adoc

```diff
@@ -1114,6 +1114,39 @@ This example defines no affinity groups:
       affinityGroupNames: []
 ----
 |String
+|`<machine-pool>.platform.ovirt.AutoPinningPolicy`
+| Optional. AutoPinningPolicy defines the policy to automatically set the CPU and NUMA settings including pinning to the host for the instance. When the field is omitted, the default is `none`. Supported values: `none`, `resize_and_pin`.
+
+[source,yaml]
+----
+<machine-pool>:
```

Comment:
Is this a section of a YAML file, or the entire file? Use commented ellipses and verify indentation if former.  

Leading the example with `<machine-pool>` gives me pause. Could you avoid using a fill-in by contextualizing the example, e.g., "For example, for a compute machine pool..."?